src/app/query-explainer/query-explainer.service.spec.ts

```diff
@@ -46,7 +46,7 @@ describe("QueryExplainerService", () => {
         const errorMessage = "Error";
         const toastrServiceSpy = spyOn(toastrService, "error");
 
-        service.processQuery(mockQuery).subscribe({
+        service.processQuery(mockQuery, ["Proof"]).subscribe({
```

Comment:
Make sure that the request for data is not asking for `Proof` - we just need the data and hashing + signing to create the proof is computationally expensive.

Reply:
Fixed